cli/src/action/circuit/api.rs

```diff
@@ -257,6 +259,12 @@ impl fmt::Display for CircuitSlice {
             display_string += "Display Name: -\n    ";
         }
 
+        if let Some(status) = &self.circuit_status {
+            display_string += &format!("Circuit Status: {:?}\n    ", status);
```

Comment:
This should  match on status and output a string, otherwise, it will display the qualified variant, like `CircuitStatus::Active`

Reply:
Updated to include using a `From` implementation on the `CircuitStatus` that is displayed here (from the `admin::messages` module) to convert the full variant to just `Active/Disbanded/Abandoned`, depending.